core/templates/pages/exploration-player-page/learner-experience/conversation-skin.directive.ts

```diff
@@ -876,7 +876,7 @@ angular.module('oppia').directive('conversationSkin', [
 
             PlayerTranscriptService.addNewInput(answer, false);
 
-            if ($scope.canAskLearnerForAnswerInfo()) {
+            if ($scope.getCanAskLearnerForAnswerInfo()) {
```

Comment:
This should probably be `$scope.canAskLearnerForAnswerInfo` or the funtion name should be renamed accordingly. 

See
https://github.com/oppia/oppia/blob/bd7029c985afd3bb79535a393b851035a094acc6/core/templates/pages/exploration-player-page/learner-experience/conversation-skin.directive.ts#L416-L418

Reply:
How did you test this change btw?

Reply:
I think I didn't change some occurences of the function name. I'll change ```$scope.canAskLearnerForAnswerInfo = function() { ```  to ```$scope.getCanAskLearnerForAnswerInfo = function() {``` . 

I tested the changes with the ```scripts.run_frontend_tests``` command. How should I test this service manually?

Reply:
Sounds good. Here's a video explaining how to test [stuff](https://vimeo.com/442416282). Feel free to reach out to me if you have any questions. Thanks!